packages/react-native-reanimated/src/component/PerformanceMonitor.tsx

```diff
@@ -180,11 +241,44 @@ function UiPerformance() {
   );
 }
 
-export function PerformanceMonitor() {
+const DEFAULT_EXPECTED_FPS = 60;
+const DEFAULT_BUFFER_SIZE = 20;
+
+export type PerformanceMonitorProps = {
+  /**
+   * DEFAULT: 60
+   *
+   * Sets the highest expected fps value.
+   *
+   * Affects dynamic smoothing rate, but isn't critical to component's operation.
+   */
+  expectedFps?: number;
+
+  /**
+   * DEFAULT: 20
+   *
+   * Sets amount of previous frames used for smoothing at highest expectedFps.
+   *
+   * Automatically scales down at lower frame rates.
+   *
+   * Affects jumpiness of the FPS measurements value.
+   */
+  smoothingCoefficient?: number;
+};
+export function PerformanceMonitor({
+  expectedFps,
+  smoothingCoefficient,
+}: PerformanceMonitorProps) {
   return (
     <View style={styles.monitor}>
-      <JsPerformance />
-      <UiPerformance />
+      <JsPerformance
+        expectedFps={(expectedFps ?? DEFAULT_EXPECTED_FPS) / 2} // /2 due to 2x lower sampling rate on JS due to performance issues
+        smoothingCoefficient={smoothingCoefficient ?? DEFAULT_BUFFER_SIZE}
```

Comment:
I am not sure if we just don't want a default values for the props. Then we don't need the `PerformanceMonitorProps` type and we can describe props with a TSDoc comment like here: https://github.com/software-mansion/react-native-reanimated/blob/ac8b705d1bd1ed770679e8009144254fe34bd517/packages/react-native-reanimated/src/hook/useAnimatedScrollHandler.ts#L28-L37

Reply:
Sure, I can do that.

Although `expectedFps` has a very specific use case - people testing fps on phones where they expect fps higher than 60, and in some cases higher than 120 might want to change this value to their expected fps amount, having this value either be too low or too high will only really affect the smoothing quality, which is not all that essential.

I'll keep this value at 60 by default, but will change smoothing rate to 30, and make both of these values constant by removing the props associated with them.

Changing smoothing rate to 30 will make it reasonably smooth on both 60fps `(0.5s smoothing look-back)` and 120fps `(0.25s smoothing look-back)` devices.

Reply:
Seems reasonable to me 😄